radio/src/targets/taranis/CMakeLists.txt

```diff
@@ -504,12 +504,21 @@ elseif (CPU_TYPE_FULL STREQUAL STM32F413xG)
   add_definitions(-DSTM32F40_41xxx -DSTM32F413xx)
   set(TARGET_LINKER_DIR stm32f413)
 else()
-  message( FATAL_ERROR "Unknown CPU_TYPE_FULL" )
+  message(FATAL_ERROR "Unknown CPU_TYPE_FULL" )
 endif()
 
-if(TRANSLATIONS STREQUAL UA AND NOT TARGET_FLASH_SIZE STREQUAL 1M)
+if(NOT TARGET_FLASH_SIZE STREQUAL 1M AND (TRANSLATIONS STREQUAL RU OR TRANSLATIONS STREQUAL UA))
+if(((PCB STREQUAL X9D+) AND (PCBREV STREQUAL 2019)) OR
+   ((PCB STREQUAL X7) AND (PCBREV STREQUAL ACCESS)) OR
+   (PCB STREQUAL XLITES) OR
+   (PCB STREQUAL X9LITE) OR 
+   (PCB STREQUAL X9LITES))
+   message(FATAL_ERROR "Specified translation will not fit target without disabling internal RF" )
```

Comment:
Beside it might be obsolete because of 4827, I think  -DUSBJ_EX=NO would be a better alternative (it saves 6k and doesn't cripple main radio functionality imho)

Reply:
Yeah, the PXX2 disable isn't needed anymore... and yes, that would be a better idea... I won't ask why that wasn't done though :zany_face: Given there is a smidgen over 1K remaining,the X9D+2019 may have company in the future for turning that option off.